tealer/utils/output.py

```diff
@@ -417,6 +416,58 @@ def write_to_files(self, dest: Path) -> bool:
         return False  # this statement is needed for darglint
 
 
+class InstructionsOutput(Output):
+    def __init__(
+        self, teal: "Teal", detector: "AbstractDetector", instructions: List["Instruction"]
+    ):
+        self._teal = teal
+        self._detector = detector
+        self.instructions: List["Instruction"] = instructions
+
+    @property
+    def detector(self) -> "AbstractDetector":
+        return self._detector
+
+    def filter_paths(self, filter_regex: str) -> None:
+        pass
+
+    def to_json(self) -> Dict:
+        result = {
+            "type": "InstructionsOutput",
+            "count": len(self.instructions),
+            "description": detector_terminal_description(self.detector),
+            "check": self.detector.NAME,
+            "impact": str(self.detector.IMPACT),
+            "confidence": str(self.detector.CONFIDENCE),
+            "help": self.detector.WIKI_RECOMMENDATION.strip(),
+            "paths": [str(ins) for ins in self.instructions],
+        }
+        return result
+
+    def write_to_files(self, dest: Path) -> bool:
```

Comment:
As you suggested, its better to change this to more generic name `ui_output` or `output` something.